python/demo/poisson/demo_poisson.py

```diff
@@ -76,30 +76,37 @@
 import numpy as np
 
 import ufl
-from dolfinx import fem, plot
-from dolfinx.fem import FunctionSpace, dirichletbc, locate_dofs_topological
-from dolfinx.io import XDMFFile
-from dolfinx.mesh import CellType, create_rectangle, locate_entities_boundary
-from ufl import ds, dx, grad, inner
-
+from dolfinx import fem, plot, io, mesh as _mesh
 from mpi4py import MPI
 from petsc4py.PETSc import ScalarType
 
 # We begin by defining a mesh of the domain and a finite element
-# function space :math:`V` relative to this mesh. As the unit square is
-# a very standard domain, we can use a built-in mesh provided by the
-# class :py:class:`create_unit_square_mesh
-# <dolfinx.mesh.create_unit_square_mesh>`. In order to create a mesh
-# consisting of 32 x 32 squares with each square divided into two
-# triangles, we do as follows ::
-
-# Create mesh and define function space
-mesh = create_rectangle(MPI.COMM_WORLD, ((0.0, 0.0), (2.0, 1.0)), (32, 16), CellType.triangle)
-V = FunctionSpace(mesh, ("Lagrange", 1))
+# function space :math:`V` relative to this mesh. We create a
+# rectangular mesh using built-in function provided by
+# class :py:class:`create_rectangle<dolfinx.mesh.create_rectangle>`.
+# In order to create a mesh consisting of 32 x 16 squares with each
+# square divided into two triangles, we do as follows ::
+
+# Create an MPI communicator
+comm = MPI.COMM_WORLD
```

Comment:
I don't feel that this extra assignment add much.

On a technical level, we're not creating a communicator here. `MPI.COMM_WORLD` already exists.